src/content/docs/en/install-and-setup.mdx

```diff
@@ -172,11 +177,7 @@ Explore the guides below to customize your development experience.
     description="Set and use environment variables in your project."
     href="/en/guides/environment-variables/"
   />
-    <LinkCard
-    title="Dev Toolbar"
-    description="Explore the helpful features of the dev toolbar."
-    href="/en/guides/dev-toolbar/"
-  />
+
```

Comment:
The newline
![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExbG9lOW1wNGt0NmJjcmM3bjVxNDRidHJlZXg4amdvNXI0ODVsOWc4cCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/j2HY8kKj1A4kmu4wMO/giphy.webp)

```suggestion
```